misc.py

```diff
@@ -101,7 +98,6 @@ class Colors:
 
 
 class RenderPriority(Enum):
-
```

Comment:
Do you have an automatic formatter or something? Is this non-PEP8 compliant?

Reply:
I do have an automatic formatter. Guess it follows [these](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) guidelines.